README.md

```diff
@@ -36,6 +36,14 @@ Searches [pub.dev](https://pub.dev).
 
 ### view
 
+*Aliases: updates, u*
```

Comment:
nit: Technically "updates" isn't an alias here.

Reply:
oops, sorry thats a copy-paste typo, will fix that.